### DISCUSSION TITLE
Setup with webpack

### DISCUSSION THREAD

**Discussion by maxvia37** [lingui/js-lingui#1931] [Q&A]
Hi,
I am trying to install Lingui.js with my Electron-forge / React / Typescript project. I am using webpack. Do you know how to set up Lingui.js in that configuration? Any example will be helpful!
Thank you !

**Comment by timofei-iatsenko** [comment]
Hey, you don't need anything special to run it with webpack. Use one of the tutorials or examples in the repo. 

You need to decide what technology you would you use for transpiling your code, SWC or Babel, depending on that you need to install lingui macro, and that's it!
